modules/gentemplates/rapidml-jsonschema/src/test/java/com/reprezen/genflow/rapidml/jsonschema/xchange/realizations/LinkSpec_CollectionResource_Test.xtend

```diff
@@ -17,8 +17,7 @@ class LinkSpec_CollectionResource_Test extends RealizationTestBase {
 			'''
 				type: object
 				minProperties: 1
-				description: "The supporting data type for the Index resource.  Not meaningful
-				  as a \nbusiness entity, but required to support a single point of entry."
+				description: "The supporting data type for the Index resource.  Not meaningful as a business entity, but required to support a single point of entry."
```

Comment:
The test was still failing because the model file was not updated. I did it in 
https://github.com/RepreZen/GenFlow/pull/45/commits/12b71c28546a1e38c5258bebe4a9a34b12b26b0f